cli/tools/lint/reporters.rs

```diff
@@ -209,13 +211,21 @@ impl LintReporter for JsonLintReporter {
       code: d.code().to_string(),
       hint: d.hint().map(|h| h.to_string()),
     });
+
+    if !self.checked_files.contains(&d.specifier.to_string()) {
+      self.checked_files.push(d.specifier.to_string());
```

Comment:
To have the consistent format this should probably do `d.specifier.to_file_path()`